tests/src/errors.rs

```diff
@@ -18,7 +18,7 @@ fn invalid_pending_frame_count() {
 
 #[test]
 fn end_frame_unclosed_query() {
-    let (_, device, _) = create_device(wgpu::Features::TIMESTAMP_QUERY).unwrap();
+    let (_, device, _queue) = create_device(wgpu::Features::TIMESTAMP_QUERY).unwrap();
```

Comment:
Believe it or not, the test was panicking without this change, since the queue was getting dropped before calling Device::create_command_encoder. I added a test to wgpu to check for this: 

https://github.com/gfx-rs/wgpu/pull/5211

Reply:
huh interesting. wonder why I didn't hit this before. Thanks for following up over on wgpu!